irrep/spacegroup.py

```diff
@@ -658,35 +720,103 @@ def __init__(
+                self.name = dataset['international']
+                self.number = dataset['number']
+                refUC_tmp = dataset['transformation_matrix']
+                shiftUC_tmp = dataset['origin_shift']
+                rotations = dataset['rotations']
+                translations = dataset['translations']
+            else:
+                self.name = dataset.international
+                self.number = dataset.number
+                refUC_tmp = dataset.transformation_matrix
+                shiftUC_tmp = dataset.origin_shift
+                rotations = dataset.rotations
+                translations = dataset.translations
+            time_reversal_list = [False] * len(rotations)
+
+        else:  # Magnetic moments
+
+            self.magnetic = True
+            dataset = spglib.get_magnetic_symmetry_dataset((*cell,
+                                                            magmom))
+            if dataset is None:                                                 
+                raise ValueError("No magnetic space group could be detected!")  
+            rotations = dataset.rotations
+            translations = dataset.translations
+            time_reversal_list = dataset.time_reversals
+            refUC_tmp = dataset.transformation_matrix
+            shiftUC_tmp = dataset.origin_shift
+
+            uni_number = dataset.uni_number
+            root = os.path.dirname(__file__)                                    
+            with open(root + "/msg_numbers.data", 'r') as f:                    
+                self.number, self.name = f.readlines()[uni_number].strip().split(" ") 
+
+        # Read syms from .sym file (useful for Wannier interface)
+        if from_sym_file is not None:
+            assert alat is not None, "Lattice parameter must be provided to read symmetries from file"
+            rot_cart, trans_cart = read_sym_file(from_sym_file)
+            rotations, translations = cart_to_crystal(rot_cart,
+                                                      trans_cart,
+                                                      self.Lattice,
+                                                      alat)
+            translation_mod_1 = False
+        else:
+            translation_mod_1 = True
+
+        symmetries = []
+        for isym, rot in enumerate(rotations):
+            symmetries.append(SymmetryOperation(rot,
+                                                translations[isym],
+                                                self.Lattice,
+                                                ind=isym+1,
+                                                spinor=self.spinor,
+                                                translation_mod1=translation_mod_1,
+                                                time_reversal=time_reversal_list[isym]))
+
+        if self.magnetic:
+            self.symmetries = list(filter(lambda x: not x.time_reversal,
```

Comment:
I would prefer that the list "symmetries" contains ALL symmetries - both unitary and antiunitary.

If you need them separated, you may create two filtered lists:
```python
self.u_symmetries = ...
self.au_symmetries = ...
```

Reply:
I will take care of it.

Reply:
We are currently testing. @MIraola has already made the requested changes and I have refactored the `Spacegroup.match_symmetries` method to make it cleaner. I have also added a `--time-reversal` flag to the CLI that makes `irrep` interpret the system as having TRS and using the appropriate magnetic space group.